managed/services/agents/vmagent.go

```diff
@@ -16,19 +16,31 @@
 package agents
 
 import (
+	"os"
 	"sort"
 
 	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/api/inventorypb"
 )
 
+var (
+	maxScrapeSizeEnv     = "PMM_PROMSCRAPE_MAX_SCRAPE_SIZE"
```

Comment:
not sure if we have agreement on how we name env vars for config like this
can you suggest what it is better to put in pmm doc?
@BupycHuk @artemgavrilov

Reply:
I don't know any conventions about env vars naming. You variant looks good for me, probably it can be even simplified to `PMM_MAX_SCRAPE_SIZE`